default/scripting/buildings/ABANDON_OUTPOST.focs.txt

```diff
@@ -62,6 +62,15 @@ BuildingType
                     empire = Source.Owner
                 SetVisibility empire = Source.Owner visibility = Max(Value, Partial)
             ]
+         EffectsGroup
```

Comment:
does this need to be a separate effects group?

Reply:
> does this need to be a separate effects group?

I do not want to add SitReps for the building, plus Target.ID wouldn't be a planet-ID for the buildings, so the additional SitReps would be broken. Or the stackinggroup would prevent the effect to be executed more than once, not sure what it is good for in the planet effect. I don't think we can create one SitRep, but change multiple objects in one EffectGroup, can we?

Reply:
one can with If/else effects, but it's not the preferred implementation